serialization_spec/serialization.py

```diff
@@ -113,7 +118,7 @@ def make_serializer_class(model, serialization_spec):
                 {'model': model, 'fields': get_fields(serialization_spec)}
             ),
             **{
-                key: SerializerLambdaField(impl=lambda value: [str(each.id) for each in getattr(value, key).all()])
+                key: SerializerLambdaField(impl=make_id_list_getter(key))
```

Comment:
This fix is very verbose (though now correct). Do you think perhaps the whole idea of `SerializerLambdaField(impl=lambda...` is a bad one? If feels like its just asking to be used wrong...

Can you think of a better/more pythonic way to implement this idea than using an impl=lambda?